geopandas/base.py

```diff
@@ -391,6 +391,46 @@ def is_ring(self):
         """
         return _delegate_property("is_ring", self)
 
+    @property
+    def is_ccw(self):
+        """Returns a ``Series`` of ``dtype('bool')`` with value ``True``
+        if a linestring or linearring is counterclockwise.
+
+        Note that there are no checks on whether lines are actually
+        closed and not self-intersecting, while this is a requirement
+        for ``is_ccw``. The recommended usage of this function for
+        linestrings is ``is_ccw(g) & is_simple(g)`` and for
+        linearrings ``is_ccw(g) & is_valid(g)``.
```

Comment:
```suggestion
        for ``is_ccw``. The recommended usage of this property for
        linestrings is ``GeoSeries.is_ccw & GeoSeries.is_simple`` and for
        linearrings ``GeoSeries.is_ccw & GeoSeries.is_valid``.
```
Rewording to match GeoPandas api instead of shapely.